include/openthread/ip6.h

```diff
@@ -653,6 +653,58 @@ typedef bool (*otIp6SlaacPrefixFilter)(otInstance *aInstance, const otIp6Prefix
  */
 void otIp6SetSlaacPrefixFilter(otInstance *aInstance, otIp6SlaacPrefixFilter aFilter);
 
+/**
+ * This function pointer is called with results of `otIp6RegisterMulticastListeners`.
+ *
+ * @param[in]  aContext  A pointer to the user context.
+ * @param[in]  aError    OT_ERROR_NONE when successfully sent MLR.req and received MLR.rsp,
+ *                       OT_ERROR_BUSY when a previous registration was ongoing,
+ *                       OT_ERROR_INVALID_ARGS when the one or more arguments were invalid,
+ *                       OT_ERROR_INVALID_STATE when the device had no active Commissioner session,
+ *                       OT_ERROR_INVALID_STATE when there was no Primary Backbone Router in Thread network,
```

Comment:
Yes it is. 
We conventionally returns  `OT_ERROR_INVALID_STATE` when there is no PBBR in the network. 

I was intended to add a new `OT_ERROR_NO_PRIMARY_BACKBONE_ROUTER` error, but thought it's better added in a separate PR because other code needs to be revised accordingly too.